monkey/monkey_island/cc/environment/set_server_config.py

```diff
@@ -1,7 +1,8 @@
 import argparse
 import json
 import logging
-from pathlib import Path
+
+from monkey_island.cc.environment.environment_config import EnvironmentConfig
```

Comment:
python monkey/monkey_island/cc/environment/set_server_config.py testing

Reply:
Changing to local  probably won't work either, since EnvironmentConfig also imports, so I just changed CWD of travis to match CWD of island